README.md

```
@@ -1,66 +1,61 @@
+  <h1 align="center">AI Agent for Cloud Troubleshooting and Alert Investigation</h1>
 
-Respond to alerts faster, using AI to automatically:
+HolmesGPT is an AI agent for investigating problems in your cloud, finding the root cause, and suggesting remediations. It has dozens of built-in integrations for cloud providers, observability tools, and on-call systems.
 
-- Fetch logs, traces, and metrics
-- Determine if issues are application or infrastructure related
-- Find upstream root-causes
+>🎉 **HolmesGPT is now a CNCF Sandbox Project!** We're thrilled to be part of the Cloud Native Computing Foundation. [Learn more about our journey](https://github.com/cncf/sandbox/issues/392#issuecomment-3380007501).
 
-Using HolmesGPT, you can transform your existing alerts from this 👇
+Find more about HolmesGPT's maintainers and adopters [here](./ADOPTERS.md).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)